docs/release-notes/release-notes-0.16.0.md

```diff
@@ -303,6 +303,9 @@ in the lnwire package](https://github.com/lightningnetwork/lnd/pull/7303)
 * [Fixup GenCertPair use](https://github.com/lightningnetwork/lnd/pull/7388)
   after updating to cert/v1.2.1.
 
+* [Fix the `wait.NoError` function to correctly handle hanging 
```

Comment:
Does this belong to `0.16.1`? Or maybe use the `no-changelog` label as it's an internal fix.

Reply:
oh cool - didn't know about `no-changelog`. Will add :)